assembl/static2/js/app/client.js

```diff
@@ -40,16 +46,28 @@ const networkInterface = createNetworkInterface({
 networkInterface.use([
   {
     applyMiddleware: function (req, next) {
-      Sentry.addBreadcrumb({
-        category: 'graphql',
-        message: `GraphQL operation: ${req.request.operationName}`,
-        data: {
-          variables: req.request.variables
-        },
-        level: 'info'
-      });
+      if (!req.options.headers) {
+        req.options.headers = {}; // Create the header object if needed.
+      }
+      co(async () => {
+        if (useCSRFProtection === 'true') {
```

Comment:
depending what you decide on my comment above, this line might have to be updated